src/dex-ui/pages/dao/DAODetailsPage/MultiSigDAODashboard/Settings.tsx

```diff
@@ -0,0 +1,266 @@
+import { Button, Divider, Flex, SimpleGrid, Tag, Text, IconButton, Image } from "@chakra-ui/react";
+import { CopyIcon, DeleteIcon } from "@chakra-ui/icons";
+import { useForm } from "react-hook-form";
+import { FormInput } from "@dex-ui-components";
```

Comment:
You can condense the @dex-ui-components imports into a single line: `import { FormInput, RefreshIcon, Color } from "@dex-ui-components";`

Reply:
My bad, its done.